rootfs/start.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -z ${EXECUTE_DEV_INSTALL+x} ]; then
+    composer install --no-interaction
+    composer clear-cache
+fi
+if [ ! -z ${EXECUTE_LARAVEL_OPTIMIZATION+x} ]; then
+    php artisan config:cache
+    php artisan route:cache
+    php artisan vendor:publish --tag=public
+fi
+
+if [ ! -z ${EXECUTE_MIGRATIONS+x} ]; then
+    php artisan migrate:refresh --seed
+fi
```

Comment:
As talked before, EXECUTE_MIGRATIONS should be renamed to EXECUTE_REFRESH and add an `else` statement containing `php artisan migrate --force`

Reply:
Ok. I'll do the changes